cfme/tests/generic_objects/test_instances.py

```diff
@@ -3,17 +3,77 @@
 import fauxfactory
 import pytest
 
+
 from cfme import test_requirements
+from cfme.base.login import BaseLoggedInPage
 from cfme.services.myservice import MyService
-from cfme.utils.appliance import ViaREST
+from cfme.rest.gen_data import categories as _categories, tags as _tags
+from cfme.utils.appliance import ViaREST, ViaUI
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.rest import assert_response
 from cfme.utils.update import update
-from cfme.fixtures.pytest_store import store
 
-pytestmark = [test_requirements.generic_objects]
+
+pytestmark = [
+    test_requirements.generic_objects,
+    pytest.mark.uncollectif(lambda appliance: appliance.version < "5.9",
+                            reason="5.8 appliance doesn't support generic objects")
+]
+
+
+@pytest.yield_fixture
```

Comment:
**Required**

`yield_fixture` was deprecated, just using `fixture` decorator now.

Reply:
Note - you can still yield! Just the decorator changed.